src/Domain/composer.json

```diff
@@ -31,7 +31,7 @@
     "require-dev": {
         "doctrine/orm": "^2.6",
         "ramsey/uuid": "^3.7",
-        "symfony/messenger": "^4.2",
+        "symfony/messenger": "^4.3",
```

Comment:
can you also update the `conflict` rule above

Reply:
I've bumped it, and I think it got added correctly to the PR.  Alas, I'm not sure what the conflict rule does, so I just changed it from 4.2 to 4.3.  

Travis is still complaining, though it's on Code Quality, and it seems to be related to self and DoctrineCollection.

Reply:
see https://getcomposer.org/doc/04-schema.md#conflict

so we ensure you cant use msgphp with messenger lower than 4.3, in your app. Our require-dev section does not apply when installing msgphp as a dependency in your app, hence the conflict rule